### PR TITLE
Feature/asm 3

### PR DESCRIPTION
**Pull request by aaronmallen**

### CODE REVIEW

File: app/controllers/clients_controller.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def set_client
     end
 
     def client_params
-      params.require(:client).permit(:name, :address, :phone, :website, :short_code)
+			params.require(:client).permit(:name, :address, :phone, :website, :short_code, :account_manager)
```
Comment (wasi) [inline app/controllers/clients_controller.rb]: make a post request (create). pass account_manager and test, that it is saved / exists (https://github.com/asm-products/mana-rails/blob/develop/spec/controllers/clients_spec.rb#L33)

File: app/models/client.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ def to_param #override
   phony_normalize :phone
 
   belongs_to :team
+	belongs_to :account_manager, class_name: "User"
```
Comment (wasi) [inline app/models/client.rb]: https://github.com/asm-products/mana-rails/blob/develop/spec/models/membership_spec.rb#L4

File: app/views/clients/_client_form.html.slim [hunk 3]
```diff
@@ -18,6 +18,11 @@
 		= f.label :website
 		= f.text_field :website, class: "form-control", placeholder: "Website url"
 		= errors_for @client, :website
+	
+	.form-group
+		= f.label :account_manager
+		= f.collection_select :account_manager_id, @current_team.users, :id, :name, {},  { :class => 'form-control' }
+		= errors_for @client, :account_manager_id
```
Comment (wasi) [inline app/views/clients/_client_form.html.slim]: fill in the form via request spec
https://github.com/asm-products/mana-rails/blob/develop/spec/features/user_request_spec.rb#L8

**Comment by aaronmallen** [comment]
@wasi submitted requested specs, with the exception of the client_request_spec.  I left what I did commented out so that I could get your help on it later.

**Pull request merged**